reviews two/swagger.yaml

```diff
@@ -0,0 +1,879 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Draft Orders API
+  description: >
+    ### Draft Orders API 
+      
+      This API is about operations you can perform on the draft order object as defind below:
+    
+    Field | Definition | Data Type
+      -----|-----------|-------------------------
+      `id`| `id` of the draft order |`string`
+      `order_id`  | The id of the order associated to the draft order, once created. | `string`        
+      `name` | Name of the draft order, format #D<number>, where number, is an sequential identifier unique to the shop, starting at 1. For example #D133 | `integer`
+      `customer` | Customer object will be serialized with only the default address, however only the ID can be set in order to associate the customer to the draft order. Setting the value to null removes the customer from the draft order. | `object`
+      `shipping_address` | The mailing address to where the draft order will be shipped. | `string`
+      `billing_address` | The mailing address associated with the payment method. | `string`
+      `note` | The text of an optional note that a shop owner can attach to the draft order. | `string`
+      `email` | The email address used for sending notifications. |`string`
+      `currency` | The three letter code for the currency to be used for the payment. | `string`
+      `invoice_sent_at` | DateTime when the invoice was emailed to the customer by Shopify. | `dateTime`
+      `invoice_url` | The url to send to the customer so that they can complete the checkout.  When using `send_invoice`, this url is emailed to the customer. This field can be used so that an API client can use another method of communication to provide the url to the customer. | `string`
+      `line_item`[ ] | | array of `line_item` objects
+      `metafields`[ ] | | array of `metafield` objects
+      `shipping_line` | | `object`
```

Comment:
Also missing description.
An object which details the shipping rate which has been selected for the draft order. The shipping line is automatically reset to null when the applied shipping rate is no longer valid.